### PR TITLE
fencing: Add Error Message & Use EC_OK

### PR DESCRIPTION
**Pull request by razo7**
~~- Four error codes (EC_INVALID_PRIVILEGES, EC_STATUS_HMC, EC_FETCH_VM_UUID, and EC_BAD_ARGS) have not been used, thus they can be dismissed.~~
- Add error message for EC_GENERIC_ERROR
- Use EC_OK and return result (equals to EC_OK).

### CODE REVIEW

**Comment by oalbrigt** [comment]
We want to keep those. They could be used by external agents, and also we dont want to change the return code numbers as pcs and other utilities map their values to know what has happened.

**Comment by razo7** [comment]
I understand. What about the `Add error message for EC_GENERIC_ERROR` commit? Should it stay with EC_OK/result commit?

**Comment by oalbrigt** [comment]
Yeah. That looks good to me. I dont think it was intended for that, but shouldnt hurt.

File: lib/fencing.py.py [hunk 1]
```diff
@@ -573,6 +573,7 @@ def fail_usage(message="", stop=True):
 
 def fail(error_code, stop=True):
 	message = {
+		EC_GENERIC_ERROR : "Failed: Generic error"
```
Comment (oalbrigt) [inline lib/fencing.py.py:576]: You forgot a comma at the end of the line.

**Comment by oalbrigt** [comment]
Thanks.

**Pull request merged**